src/ert/gui/main.py

```diff
@@ -40,8 +40,14 @@ def run_gui(args: Namespace, plugin_manager: ErtPluginManager | None = None) ->
     signal(SIGINT, SIG_DFL)
 
     QDir.addSearchPath("img", str(files("ert.gui").joinpath("resources/gui/img")))
-
+    qcolor_space_category = QLoggingCategory(
+        "QColorSpace"
+    )  # This category is to remove noise from the QColorSpace warnings on QApplication creation
```

Comment:
This name does not matter at all, it does not need to match the warning.
Also, maybe we should comment which warning we aim to suppress.